### ISSUE TITLE
Create script for updating theme

### ISSUE THREAD

**Issue by mattstratton** [devopsdays/devopsdays-web#1846]
Once we release `devopsdays-theme`, the mechanism for updating it in this repo is going to be by downloading the artifact from https://github.com/devopsdays/devopsdays-theme/releases, and then updating `themes/devopsdays-theme` with the contents of that archive.

I am suggesting we create a script to automate most of this - basically, my theory is that you run it from within this repo, it copies down the zip file, extracts it to a temporary location, blows away `themes/devopsdays-theme`, and then copies the zip file stuff into it. Then it deletes the temporary directory, does a `git add themes/devopsdays-theme`, and then you can do you own PR against it.

Basically, I'll end up writing this, but I don't want to lose track of tracking this.

**Comment by mattstratton** [comment]
Here's my minimal viable bash that does it. 

```
#!/bin/bash

if [ $# -ne 1 ]; then
    echo $0: usage: dod-release.sh version
    exit 1
fi

version=$1
branch=release-theme-version-$version
rm -rf devopsdays-web
git clone https://github.com/devopsdays/devopsdays-web.git
cd devopsdays-web
git checkout -b $branch
wget https://github.com/devopsdays/devopsdays-theme/releases/download/$version/devopsdays-theme-$version.zip
rm -rf themes/devopsdays-theme
unzip devopsdays-theme-$version.zip -d themes/devopsdays-theme
rm devopsdays-theme-$version.zip
git add themes/devopsdays-theme
git commit -m "Releasing theme version $version"
git push origin $branch
hub pull-request -m "Updating theme to $version"
```

This requires that you have [hub](https://github.com/github/hub) installed on your system.

**Comment by mattstratton** [comment]
I think if I add an argument about WHERE you want this to occur (it needs to be in a temporary space because it has destructive things to the local filesystem), I can probably put this up in `utilities`.

@bridgetkromhout I am open to your review.
